dcrwallet.go

```diff
@@ -144,6 +139,7 @@ func walletMain() error {
 			startPromptPass(w)
 		}
 		startWalletRPCServices(w, rpcs, legacyRPCServer)
+		startTicketBuyerService(rpcs, loader, &cfg.tbCfg)
```

Comment:
this function can be removed and it should be performed in startWalletRPCServices.